SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorCloudLauncherSettings.h

```diff
@@ -0,0 +1,163 @@
+	FFilePath PrimaryLaunchConfigPath;
+
+	UPROPERTY(EditAnywhere, config, Category = "General", meta = (ConfigRestartRequired = false, DisplayName = "Snapshot path"))
+	FFilePath SnapshotPath;
+
+	UPROPERTY(EditAnywhere, config, Category = "General", meta = (ConfigRestartRequired = false, DisplayName = "Region"))
+	TEnumAsByte<ERegionCode::Type> PrimaryDeploymentRegionCode;
+
+	UPROPERTY(EditAnywhere, config, Category = "Simulated Players", meta = (ConfigRestartRequired = false, DisplayName = "Include simulated players"))
+	bool bSimulatedPlayersIsEnabled;
+
+	UPROPERTY(EditAnywhere, config, Category = "Simulated Players", meta = (EditCondition = "bSimulatedPlayersIsEnabled", ConfigRestartRequired = false, DisplayName = "Deployment mame"))
+	FString SimulatedPlayerDeploymentName;
+
+	const FString SimulatedPlayerLaunchConfigPath;
+
+	UPROPERTY(EditAnywhere, config, Category = "Simulated Players", meta = (EditCondition = "bSimulatedPlayersIsEnabled", ConfigRestartRequired = false, DisplayName = "Number of simulated players"))
+	uint32 NumberOfSimulatedPlayers;
+
+	UPROPERTY(EditAnywhere, config, Category = "Simulated Players", meta = (EditCondition = "bSimulatedPlayersIsEnabled", ConfigRestartRequired = false, DisplayName = "Region"))
+	TEnumAsByte<ERegionCode::Type> SimulatedPlayerDeploymentRegionCode;
+
+	static bool IsAssemblyNameValid(const FString& Name);
+	static bool IsProjectNameValid(const FString& Name);
+	static bool IsDeploymentNameValid(const FString& Name);
+	static bool IsRegionCodeValid(const ERegionCode::Type RegionCode);
+
+public:
+	FString GetProjectNameFromSpatial() const;
+
+	void SetPrimaryDeploymentName(const FString& Name);
+	FORCEINLINE FString GetPrimaryDeploymentName() const
+	{
+		return PrimaryDeploymentName;
+	}
+
+	void SetAssemblyName(const FString& Name);
+	FORCEINLINE FString GetAssemblyName() const
+	{
+		return AssemblyName;
+	}
+
+	void SetProjectName(const FString& Name);
+	FORCEINLINE FString GetProjectName() const
+	{
+		return ProjectName;
+	}
+
+	void SetPrimaryLaunchConfigPath(const FString& Path);
+	FORCEINLINE FString GetPrimaryLanchConfigPath() const
+	{
+		const USpatialGDKEditorSettings* SpatialEditorSettings = GetDefault<USpatialGDKEditorSettings>();
+		return PrimaryLaunchConfigPath.FilePath.IsEmpty() 
+			? SpatialEditorSettings->GetSpatialOSLaunchConfig()
+			: PrimaryLaunchConfigPath.FilePath;
+	}
+
+	void SetSnapshotPath(const FString& Path);
+	FORCEINLINE FString GetSnapshotPath() const
+	{
+		const USpatialGDKEditorSettings* SpatialEditorSettings = GetDefault<USpatialGDKEditorSettings>();
+		return SnapshotPath.FilePath.IsEmpty()
+			? FPaths::Combine(SpatialEditorSettings->GetSpatialOSSnapshotFolderPath(), SpatialEditorSettings->GetSpatialOSSnapshotFile())
+			: SnapshotPath.FilePath;
+	}
+
+	void SetPrimaryRegionCode(const ERegionCode::Type RegionCode);
+	FORCEINLINE FText GetPrimaryRegionCode() const
+	{
+		UEnum* pEnum = FindObject<UEnum>(ANY_PACKAGE, TEXT("ERegionCode"), true);
```

Comment:
```suggestion
		UEnum* Region = FindObject<UEnum>(ANY_PACKAGE, TEXT("ERegionCode"), true);
```

Reply:
Also this could be done after the `IsRegionCodeValid` check

Reply:
Renamed the variable and moved it after the check 👍.